lib/api/objectDelete.js

```diff
@@ -176,7 +179,7 @@ function objectDelete(authInfo, request, log, cb) {
                         return services.deleteObject(bucketName, objectMD,
                             objectKey, delOptions, log, (err, delResult) =>
                             next(err, bucketMD, objectMD, delResult,
-                                deleteInfo));
+                                deleteInfo), isExpiration);
```

Comment:
i am wondering about this: since `services.deleteObject()` takes a bool, this looks somewhat inconsistent between the different "branches" of this function : when doing a delete, we delegate the computation of originOp to lower layer (delete case) or do it here (delete marker).

I guess some consistency would be better, but it means either reverting to the previous code or digging deeper here...